frontend/Search.vue

```diff
@@ -193,6 +240,18 @@ export default {
         } else {
             this.query = this.$STRINGS.QUERY_DEFAULT;
         }
+        if (localStorageEnabled && localStorage.database) {
+            this.database = JSON.parse(localStorage.database);
+        }
+        if (localStorageEnabled && localStorage.databases) {
+            this.databases = JSON.parse(localStorage.databases);
+        }
+        if (localStorageEnabled && localStorage.iterativeSearch) {
+            this.iterativeSearch = JSON.parse(localStorage.iterativeSearch);
+        }
+        if (localStorageEnabled && localStorage.taxFilter) {
+            this.taxFilter = JSON.parse(localStorage.taxFilter);
+        }
```

Comment:
These should be set somewhere else, they were removed in the last refactoring

Reply:
done